currentRangesTest.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import currentRanges
+
+class CurrentRangesTest(unittest.TestCase):
+  def test_count_min(self):
+    self.assertTrue(currentRanges.count_min([4,4,5,5,4,8]) == [4,3])
```

Comment:
One of the questions to ask when we write a test is: can this serve as an example for others who may want to understand /call this function? In the returned array [4,3], it's hard to see which is the value and which is th count.
If we realise this as we write the test, we could decide to make it a dictionary.